book/Dockerfile

```diff
@@ -1,13 +1,18 @@
 FROM rust:alpine as builder
-LABEL stage=code-library-intermediate
-WORKDIR /usr/src/code-library-book
-COPY . .
+
 RUN apk add gcc musl-dev postgresql-dev
 ENV RUSTFLAGS="-C target-feature=-crt-static"
-RUN cargo install --path .
+
+WORKDIR /usr/src
+COPY . .
+
+RUN cargo install --path book/
+
 
 FROM alpine:latest
-RUN apk add libgcc libpq
+RUN apk --no-cache add libgcc libpq
+
 COPY --from=builder /usr/local/cargo/bin/book /usr/local/bin/book
+
 EXPOSE 8080/tcp
-CMD ["book"]
+CMD [ "book" ]
```

Comment:
Shouldn't this be `ENTRYPOINT [ "book" ]`?

Reply:
True